takwimu/takwimu_ui/src/Theme.js

```diff
@@ -114,6 +114,10 @@ const Theme = createMuiTheme({
     MuiButton: {
       color: 'primary',
       variant: 'contained'
+    },
+    MuiPaper: {
+      elevation: 0,
```

Comment:
I think this might cause problems down with other pop ups such as profile detail search and releases where we need them to have a shadow.

And this
https://github.com/TakwimuAfrica/Takwimu/pull/489/files#diff-d4778892734c83ea1c6e5bbed582942aR156

Reply:
Takwimu UX is flat. It doesn't use shadows (`elevation` is implemented as shadow). Even in dropdowns like *Data by topic*.
![Screenshot from 2019-04-04 13-43-41](https://user-images.githubusercontent.com/1779590/55550002-d99ab000-56df-11e9-8542-48b6674c0367.png)

If there is a component that needs shadow, then that's an exception and it should by added to that component per design @karimkawambwa.

But as a Theme, no shadows.